mission_control/src/mission.cpp

```diff
@@ -73,6 +74,7 @@ class MissionBehaviorTreeFactory : public BT::BehaviorTreeFactory
     this->registerNodeType<mission_control::AttitudeServoBehavior>("AttitudeServoBehavior");
     this->registerNodeType<mission_control::AltitudeHeadingBehavior>("AltitudeHeadingBehavior");
     this->registerNodeType<mission_control::PayloadCommandBehavior>("PayloadCommandBehavior");
+    this->registerNodeType<mission_control::AbortBehavior>("AbortBehavior");
```

Comment:
@patapufate nit: I'd call it `Abort`. It reads better in the XML.

Reply:
Done -> 9aa5771